client/aci/client_test.go

```diff
@@ -303,21 +303,23 @@ func TestCreateContainerGroupWithBadVNetFails(t *testing.T) {
 					},
 				},
 			},
-			NetworkProfile: &NetworkProfileDefinition{
```

Comment:
seems all NetworkProfile related codes have been deleted, would this cause issues?

Reply:
As I mentioned in RP's description, the NetworkProfile has been deprecated in ACI's new API version 2021-07-01